### PR TITLE
added: superscript and subscript snippets

### PR DESCRIPTION
**Pull request by razzat008**
Added superscript and subscript snippets for markdown.
Ranging from line number `380-393`.

```diff
- }
+ },
+  "Insert subscript": {
+    "prefix": "sub",
+    "body": [
+      "${1}<sub>${0}"
+    ],
+    "description": "Create a subscript."
+  },
+  "Insert superscript": {
+    "prefix": "sup",
+    "body": [
+      "${1}<sup>${0}"
+    ],
+    "description": "Create a superscript."
+  }
```

### CODE REVIEW

**Review by OkelleyDevelopment — APPROVED** [review]
Good stuff :sunglasses:

File: snippets/markdown.json [hunk 1]
```diff
@@ -376,5 +376,19 @@
...
     "description": "Insert table with 5 rows and 5 columns. First row is heading."
+  },
+  "Insert subscript": {
+    "prefix": "sub",
+    "body": [
+      "${1}<sub>${0}"
+    ],
+    "description": "Create a subscript."
+  },
+  "Insert superscirpt": {
```
Comment (OkelleyDevelopment) [inline snippets/markdown.json:387]: nit: typo on 'superscript'

**Review by OkelleyDevelopment — APPROVED** [review]

File: snippets/markdown.json (hunk 1 above)
Comment (OkelleyDevelopment) [reply snippets/markdown.json:387]: I noticed this after approving the first time, will wait a bit to merge so you can fix it

Comment (razzat008) [reply snippets/markdown.json:387]: Fixed the typo. :)

**Pull request merged**